package.json

```diff
@@ -53,6 +53,7 @@
     "debounce": "^1.2.1",
     "debounce-stream": "^2.0.0",
     "dnode": "^1.2.2",
+    "ethereumjs-util": "^7.1.5",
```

Comment:
Looks like this package has been deprecated according to repo https://github.com/ethereumjs/ethereumjs-util. As I understand it's moved here https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util. Can we use the new one instead?

Reply:
No, we can't. `npm install @ethereumjs/util` throws an error and https://www.npmjs.com/package/@ethereumjs/util — not found

Reply:
Ok, no questions then